packages/terra-content-container/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Footer to allow sticky bottom-aligned content such as buttons, input, etc.
```

Comment:
nit: `footer` prop 

- IMO I wouldn't specify the type of content to be used as content as the proptype allows for anything

Reply:
Just to clarify, this is the changelog, not the prop docs. Regardless, I updated this line and removed content type examples.

Reply:
Nope but this is a quick reference for people that may be bumping vesions